### PR TITLE
Added image handler

### PR DESCRIPTION
**Pull request by josipslavic**
Serves images from a public folder, also added a default image into the public folder that is used for articles that are populated from the NewsAPI that don't have an image.

### CODE REVIEW

File: src/controllers/news.controller.ts [hunk 1]
```diff
@@ -48,7 +48,10 @@ export class NewsController {
     }
 
     const news = await this.newsService.createNews(
-      { ...req.body, imageUrl: req.file.path },
+      {
+        ...req.body,
+        imageUrl: `${process.env.BACKEND_URL}/public/${req.file.filename}`,
```
Comment (JCukovic) [inline src/controllers/news.controller.ts]: what will happen if we restart backend at some point? (Consider scenario where backend is running for a month in live environment, and now we need to push new version of code - let's say some new feature is added. What will happen to these images?)

**Review by JCukovic — CHANGES_REQUESTED** [review]

**Review by JCukovic — CHANGES_REQUESTED** [review]
One change needed

File: src/index.ts [hunk 2]
```diff
@@ -5,11 +5,18 @@ import { newsRouter } from './routes/news';
 import { commentRouter } from './routes/comment';
 import { swaggerDocs } from './swagger-docs';
 import mongoose from 'mongoose';
+import cloudinary from 'cloudinary';
 import 'dotenv/config';
 
 const app = express();
 const port = 3000;
 
+cloudinary.v2.config({
```
Comment (JCukovic) [inline src/index.ts]: This is not the place for cloudinary config. Try to find more suitable location

File: src/index.ts [hunk 3]
```diff
@@ -5,11 +5,18 @@ import { newsRouter } from './routes/news';
...
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET,
+});
+
 mongoose.connect(process.env.MONGO_URI as string);
```
Comment (JCukovic) [inline src/index.ts]: Same as the comment above, also, why aren't you handling connection errors (at least to know when something went wrong)? How do you know when the db is connected?

**Review by JCukovic — CHANGES_REQUESTED** [review]

**Review by JCukovic — APPROVED** [review]

File: .env.example [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# URL to the backend's index page
+BACKEND_URL=http://localhost:3000
```
Comment (anteljubic) [inline .env.example]: This env variable isn't used anywhere so it can be removed

File: readme.md [hunk 5]
```diff
@@ -33,11 +33,15 @@ This project uses the following environment variables:
 
 | Name             | Description                     | Default Value                                                                           |
 | ---------------- | ------------------------------- | --------------------------------------------------------------------------------------- |
+| BACKEND_URL      | URL to the backend's index page | http://localhost:3000                                                                   |
```
Comment (anteljubic) [inline readme.md]: You are not using this, it can be removed

**Review by anteljubic — APPROVED** [review]
Just remove not needed env variable and you can merge this

**Pull request merged**